CRM/Core/EntityTokens.php

```diff
@@ -688,4 +688,15 @@ protected function getCacheKey(): string {
     return $cacheKey;
   }
 
+  /**
```

Comment:
This function already exists at 154

Reply:
isn't that only in master?

Reply:
Right, didn't notice this was 5.65

Reply:
@larssandergreen it will merge-conflict when we try to up-merge but ....